### PR TITLE
VETTING: by phone, when disabled the information text "settings" is linked to settings page

### PR DESCRIPTION
**Pull request by eunjuhuss**
#### Description:
When "vetting by phone" is disabled and the user is getting information(description) text about how to enable "vetting by phone".
When user is missed "phone number" or has not "verified phone number", the information text will show eg, Confirm your phone number in settings.
The text "settings" is linked to the settings page and the Phone section to direkt the user to the correct section for solving the issue.

#### Summary:
- Install `npm install --save react-router-hash-link` : to navigate to specific section (phone) 
- I have split the text to enable linking of the relevant part of the text.
- I have added styling for the liked text: border-bottom black solid 1px and when hovering color changed to orange.
- I have added comments to the code: for better understanding of the code (as discussed in Slack with Fredrik)

 **don't forget to run: npm I**

---
- Disabled button: not added phone, not verified phone number. 

Screenshot 2020-12-22 at 13 38 42

- Hover effect

Screenshot 2020-12-22 at 13 38 48

- Link to phone section in settings page

Screenshot 2020-12-22 at 13 39 09

---
#### For reviewer:
- [ ] Read the above description
- [ ] Reviewed the code changes
- [ ] Navigate to the branch (pulled the latest version)
- [ ] Run the code and been able to execute the expected function
- [ ] Check any styling on both desktop and mobile sizes

### CODE REVIEW

File: src/components/LookupMobileProofing.js [hunk 1]
```diff
@@ -1,42 +1,56 @@
...
     } else if(this.props.nonSweNumber) {
       description = (
         <div className="description">
           {this.props.translate("verify-identity.vetting_explanation_only_available_swe_number")}
         </div> 
-      ); 
+      ); /* if user verified swedish number, description text will be empty */
     } else {
       <div /> 
     }
```
Comment (njons) [inline src/components/LookupMobileProofing.js:48]: This block looks to have some repeating html (only the text seem to differ between the withoutPhoneNumber/notVerifiedNumber pair of conditions and the same for the withoutNin/nonSweNumber pair of conditions). Perhaps combining the code would make it more obvious that those specific structures are the same?

File: src/components/LookupMobileProofing.js [hunk 2]
```diff
@@ -1,42 +1,56 @@
...
+          <HashLink
+            key="1"
+            to={"/profile/settings/#phone"}
+          >
+            {this.props.translate("verify-identity.vetting_link_settings")}
+          </HashLink>
         </div> 
-      )
+      ) /* without verified phone number, description text will help the user to confirm phone number and the text "setting" is linked to the setting page phone number section */
     } else if(this.props.notVerifiedNumber) {
```
Comment (njons) [inline src/components/LookupMobileProofing.js:28]: we have previously used "unverified" to describe data that has not gone through a verification or confirmation process

File: src/login/styles/_Nins.scss [hunk 3]
```diff
@@ -30,6 +30,18 @@
       display: flex;
       justify-content: space-between;
     } 
+    & .link {
+      margin: 0.5rem 0;
+      & a span:nth-child(1){
```
Comment (njons) [inline src/login/styles/_Nins.scss:35]: is the nth-child necessary? I seem to get the same styling without it. Are there other places where we have more than one link + span in a .vetting-button?

**Review by njons — COMMENTED** [review]
In case comments in code are not visible I have added a summary of comments
- Potential repeat of html in lines 11-49 in /components/LookupMobileProofing
- Flag that previous standard to describe data that has not gone though a verification process has been "unverified"
- Question about the use of nth-child in css for the .link a span in /styles/Nins.scss

File: src/components/LookupMobileProofing.js (hunk 2 above)
Comment (eunjuhuss) [reply src/components/LookupMobileProofing.js]: I have changed the name to align with previous used wording 
Now "unverifiedNumber"

File: src/components/LookupMobileProofing.js (hunk 1 above)
Comment (eunjuhuss) [reply src/components/LookupMobileProofing.js]: I have changed accordingly, it has reduced the code and made it easier to read
Thanks

**Review by njons — APPROVED** [review]
👍

**Pull request merged**